### PR TITLE
first commit

### LINKED ISSUE #7: Code entities for donations

### PR DESCRIPTION
**Pull request by yusrahsoeker**
first commit for donations class.

### CODE REVIEW

File: src/main/java/com/foodparcel/entity/Donations.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package com.foodparcel.entity;
+
+import javafx.util.Builder;
```
Comment (NWF007) [inline src/main/java/com/foodparcel/entity/Donations.java]: I think you used the wrong Builder... this is the interface from javafx.util

File: src/main/java/com/foodparcel/entity/Donations.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
+package com.foodparcel.entity;
+
+import javafx.util.Builder;
+
+public class Donations {
```
Comment (NWF007) [inline src/main/java/com/foodparcel/entity/Donations.java]: Do you mind to change class name to Donor

File: src/main/java/com/foodparcel/entity/Donations.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+            public Builder setId(String id) {
+                this.id = id;
+                return this;
+            }
+
+            public Builder setPhoneNumber(String phoneNumber) {
+                this.phoneNumber = phoneNumber;
+                return this;
+            }
```
Comment (NWF007) [inline src/main/java/com/foodparcel/entity/Donations.java]: I think you forgot the copy method :)

**Review by NWF007 — CHANGES_REQUESTED** [review]
Small changes to be made... change name of class to donor, use the correct Builder class (not the interface from javafx) and add copy method.

File: src/main/java/com/foodparcel/entity/Donations.java (hunk 3 above)
Comment (yusrahsoeker) [reply src/main/java/com/foodparcel/entity/Donations.java]: added it

File: src/main/java/com/foodparcel/entity/Donations.java (hunk 1 above)
Comment (yusrahsoeker) [reply src/main/java/com/foodparcel/entity/Donations.java]: deleted it! all is good :)

**Review by NWF007 — APPROVED** [review]
Great thanks

**Pull request merged**